internal/syncer/connector/rdbms.go

```
@@ -0,0 +1,648 @@
+
+// Close closes the RDBMS sync session.
+func (s *rdbmsSyncSession) Close() error {
+	s.closeRows()
+	return s.db.Close()
+}
+
+// openNextQuery runs the next base query.
+func (s *rdbmsSyncSession) openNextQuery(ctx context.Context) error {
+	query := s.queries[s.queryIndex]
+	s.queryIndex++
+	rows, err := s.db.QueryContext(ctx, query)
```

golangci-lint (2.12.2)

[error] rows.Err must be checked

(rowserrcheck)